nodes/src/nodes/ml_sklearn/Instance.py

```
@@ -0,0 +1,91 @@
+        if not len(documents):
+            return
+
+        # If this is a table
+        if isTable:
+            metadata.tableId = self.tableId
+            self.tableId += 1
+
+        # Write the documents
+        self.instance.writeDocuments(documents)
+
+    def open(self, object: Entry):
```

Ruff (0.15.12)

[error] Function argument `object` is shadowing a Python builtin

(A002)